### ISSUE TITLE
Set primary display

### ISSUE THREAD

**Issue by yoshegg** [phillipberndt/fakexrandr#49]
I know that there were some discussions about this topic, but are there any news?

I'm using a Matrox TripleHead2Go and Ubuntu with Gnome, unfortunately the main display will always be the leftmost.

I also tried to just leave out fakexrandr and only use `xrandr` and its `--setmonitor` option, but without success (cf. issue #48)

**Comment by phillipberndt** [comment]
No, there's no news. You'd need to hook `XRRGetOutputPrimary` to do this ([Example of how the call is used and what applications expect to receive in return](https://gist.github.com/oranenj/3658446)), which should be straightforward. The main question is how to configure this (in the GUI and how to store the information in the config file). I won't find time for this any time soon, but feel free to open a PR.

**Comment by yoshegg** [comment]
Damn :( I'll try my best then.

**Comment by yoshegg** [comment]
Well I actually did it. :muscle: 

I forked this repo and added a new branch called `matrox`. It can be found [here](https://github.com/yoshegg/fakexrandr). I'm not sure if it's worthy to be added to the main repository, but in case someone needs this in the future, here it is.

**Comment by phillipberndt** [comment]
Cool, congrats 👍 The first version of fakexrandr was exactly such a hack, with hardcoded splits, so no worries.. let's leave this open, maybe someone will step up to make this configurable as well, and then we can include the change!
